docs/rdd-programming-guide.md

```diff
@@ -86,11 +86,9 @@ import org.apache.spark.SparkConf;
 
 <div data-lang="python"  markdown="1">
 
-Spark {{site.SPARK_VERSION}} works with Python 2.6+ or Python 3.4+. It can use the standard CPython interpreter,
+Spark {{site.SPARK_VERSION}} works with Python 2.7+ or Python 3.4+. It can use the standard CPython interpreter,
```

Comment:
Ah.... Actually, 2.7 should be the final release in Python 2 (https://www.python.org/dev/peps/pep-0404/) and I remember the tests did not pass correctly in Python 3.3 ... I believe there would not be extra changes